### PR TITLE
pull in ATO walkthrough content

### PR DESCRIPTION
**Pull request by afeld**
https://trello.com/c/69weNW7R/39-pull-in-ato-walkthrough-content

This is pulled from [a conversation](https://docs.google.com/document/d/1yJ4D2z7h2_tEr7QtWfJG9ZHAg_yt3RVcBeLDXJXLfYs/edit#heading=h.k1yxbxajr2mc) with @NoahKunin, where he walked @DavidEBest and I through the ATO process. Pulled the content into a standalone page as-is for now - next step is going through it to make it more readable, see if the information should live in different pages, etc.

### CODE REVIEW

**Comment by DavidEBest** [comment]
`control` might be a terrible term, but it is the term used everywhere. I'd prefer to introduce and acclimate the user to the terminology rather than hide it.

**Comment by wslack** [comment]
@afeld LMK if you want a proofreader on anything for this today

**Comment by afeld** [comment]
To break down this PR a bit, I `reset` it back to be the commit before I started rewriting the pages, and moved the newer changes to #121. In other words, this is back to (almost) word-for-word what @NoahKunin wrote in [the Google doc](https://docs.google.com/document/d/1yJ4D2z7h2_tEr7QtWfJG9ZHAg_yt3RVcBeLDXJXLfYs/edit#heading=h.k1yxbxajr2mc). I say we merge this sooner than later without a lot of detailed review, since this is our best known "source of truth". We can then do smaller PRs on top of it.

@wslack Sure, thanks!

**Comment by wslack** [comment]
@afeld I think my copy-edits broke your changes in https://github.com/18F/before-you-ship/pull/121, even though I left the walkthrough change alone.

**Comment by DavidEBest** [comment]
This looks good to me. Should this be against `18f-pages-staging`/`18f-pages-internal` instead?

File: _pages/ato/levels.md [hunk 1]
```diff
@@ -17,6 +19,24 @@ For more information, see [NIST 800-18](http://csrc.nist.gov/publications/nistpu
...
+* The framework we usually use, since we must address a high water mark methodology anyway, is to ask ourselves (and the agency we are creating the system with) 3 worst case scenario questions:
+    * What is the worst possible outcome if all of the confidentiality of the system is lost?
+    * What is the worst possible outcome if all of the integrity of the system is lost?
+    * What is the worst possible outcome if all of the availability of the system is lost?
+* The answer to each question should then be interpreted in terms of impact to either the government or the public. The high water mark for either impacted party must be used.
+* The full list of data and functions in and of the system (in government parlance "mission based information types and delivery mechanisms") must be itemized in structured data. While the data types are obviously arbitrary and custom to each system we produce, the Government has a formalized data set of mission functions that should be mapped to the system via NIST 800-60.
+    * for a Rails app, this can simply be a link to the `db/schema.rb` on GitHub
```
Comment (DavidEBest) [inline _pages/ato/levels.md]: Where should this list of `mission based information types and delivery mechanisms` go? @18F/ato

Comment (afeld) [inline _pages/ato/levels.md]: Good question – I have no idea! https://github.com/18F/before-you-ship/issues/126

**Comment by afeld** [comment]
Moved to #127.

**Pull request closed without merging**